service/controller/resource/monitoring/scrapeconfigs/resource_test.go

```diff
@@ -163,44 +163,39 @@ func TestOpenStackScrapeconfigs(t *testing.T) {
 	var apps = []runtime.Object{
 		&appsv1alpha1.App{
 			ObjectMeta: v1.ObjectMeta{
-				Name:      "alice-default-apps",
-				Namespace: "org-my-organization",
```

Comment:
How is removing the namespace fixing it?

Reply:
the fake client, i guess fetching apps from default

Reply:
Except this is not fixing the test but making it less precise. What you have to do is change the namespace on the input values https://github.com/giantswarm/prometheus-meta-operator/tree/master/pkg/unittest/input instead

Reply:
Beware this might break other stuff but this is what needs to be done :)

Reply:
Some might need to stay in the default ns btw

Reply:
I fixed it